sdk/python/kfp/components/_data_passing.py

```diff
@@ -139,23 +139,24 @@ def get_canonical_type_for_type_struct(type_struct) -> str:
         return None
 
 
-def get_deserializer_code_for_type(typ) -> str:
-    try:
-        return type_name_to_deserializer.get(get_canonical_type_struct_for_type[typ], None)
-    except:
-        return None
+# Doesn't seem to be used anymore, can be removed?
```

Comment:
Can we just delete this unused code? Makes the repo much cleaner I'd say.

Reply:
Yeah I made a separate branch for contributing to `kfp` upstream, see [here](https://github.com/kubeflow/pipelines/pull/6504) and I got the go ahead to remove this. I'll actually be closing this PR and just be `cherry-picking` in the upstream change once it get's merged (hopefully today!).